client/dashboard/domains/domain-dns/index.tsx

```diff
@@ -186,6 +192,101 @@ export default function DomainDns() {
 		} );
 	};
 
+	const hasWpcomNameservers = () => {
+		if ( ! nameservers || nameservers.length === 0 ) {
```

Comment:
[nit] `! nameservers` won't happen since we're using `useSuspenseQuery()` above.